tests/yam/agama/boot_agama.pm

```diff
@@ -50,6 +50,15 @@ sub prepare_boot_params {
     }
     push @params, 'inst.register_url=' . get_var('SCC_URL') if get_var('SCC_URL') && get_var('FLAVOR') =~ /^(Online.*|agama-installer)$/;
 
+    # REPO_0 should be set everywhere where we rsync repo (aside from iso)
+    if (get_var('REPO_0') && is_leap) {
+        my $host = get_var('OPENQA_HOST', 'https://openqa.opensuse.org');
+        my $repo = get_var('REPO_0');
+
+        # inst.install_url supports comma separated list if more repos are needed ...
+        push @params, "inst.install_url=$host/assets/repo/$repo";
```

Comment:
One solution that cross my mind could be if you could pass this directly as EXTRABOOTPARAMS='inst.install_url=%OPENQA_HOST%/assets/repo/%REPO_0%' as job configuration. Disadvantage is that you would need to set it for each job using this so I don't know the scope of the change.
What I notice is that we don't need this in SLES, only for some hack in s390x, it is only a Leap thing afir.
We try to avoid condition like this:
`if (get_var('REPO_0') && is_leap) {`

Another solution could be to to create `INST_INSTALL_URL` and then you just fix the repo there in job group, per architecture for example. The less parsing, manipulation we do in this module the better and should not be leap-specific to introduce a new param.